docs-kits/kits/DCM-Kit/page_changelog.md

```diff
@@ -11,6 +11,22 @@ sidebar_position: 1
 
 All notable changes to this Kit will be documented in this file.
 
+## [0.2.0] - 2024-02-xx
```

Comment:
Please add the date of the release: 2024-03-08

Reply:
Fixed